buildroot-external/patches/occu/0187-WebUI-Fix-GroupEditingIfDevicesInInbox/occu/WebUI/www/webui/js/common/viewmodels.js

```
@@ -0,0 +1,192 @@
+    var virtualDevice = new GroupDevice(selfGroup.virtualDeviceSerialNumber,selfGroup.virtualDeviceSerialNumber);
+    var url = '/pages/tabs/control/devices.htm?sid='+SessionId;
+    var pb = "{}";
+    var opt =
+    {
+      postBody: pb,
+      onComplete: function(t)
+      {
+        WebUI.previousPage        = WebUI.currentPage;
+        WebUI.previousPageOptions = WebUI.currentPageOptions;
+        WebUI.currentPage = ControlDevicesPage;
+        var regularExpression = new RegExp("loadChannels\\W\\s[0-9]+\\s\\W");
```

Biome

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)